lightkurve/tests/test_targetpixelfile.py

```diff
@@ -207,14 +214,20 @@ def test_repr():
 
 
 def test_to_lightcurve():
-    for tpf in [KeplerTargetPixelFile(filename_tpf_all_zeros),
+    for tpf in [TargetPixelFile(filename_tpf_all_zeros),
+                KeplerTargetPixelFile(filename_tpf_all_zeros),
                 TessTargetPixelFile(filename_tess)]:
         tpf.to_lightcurve()
         tpf.to_lightcurve(aperture_mask=None)
         tpf.to_lightcurve(aperture_mask='all')
         lc = tpf.to_lightcurve(aperture_mask='pipeline')
-        assert lc.astropy_time.scale == 'tdb'
         assert lc.label == tpf.hdu[0].header['OBJECT']
+        # Generic TPFs have unknown timescales by default
+        if tpf.__class__ == TargetPixelFile:
+            assert lc.time_scale == None
+            assert lc.time_format == None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/app/barentsen/lightkurve/pullRequest?prid=4287604)